image/templates/helm/stackrox-central/values.yaml

```diff
@@ -188,8 +188,10 @@
 #
 #    # Route configuration.
 #    # Disabled by default.
+#    # Specify a custom hostname if desired, otherwise accept the default from OpenShift
```

Comment:
```suggestion
#    # Specify a custom hostname if desired, otherwise accept the default from OpenShift.
```

Also, I think this comment might fit better right before the `host` line?